forge-gui/res/cardsfolder/upcoming/picnic_ruiner_stolen_goodies.txt

```diff
@@ -0,0 +1,19 @@
+Name:Picnic Ruiner
+ManaCost:1 R
+Types:Creature Goblin Rogue
+PT:2/2
+T:Mode$ Attacks | ValidCard$ Card.Self | IsPresent$ Creature.YouCtrl+powerGE4 | TriggerZones$ Battlefield | Execute$ TrigPump | TriggerDescription$ Whenever CARDNAME attacks while you control a creature with power 4 or greater, CARDNAME gains double strike until end of turn.
```

Comment:
> If you controlled a creature with power 4 or greater when you declared Picnic Ruiner as an attacker, it doesn't matter whether you still control one as its ability resolves.

Reply:
Does this need to be done with a `checksvar` ?

Reply:
that's no difference
you need NoResolvingCheck$